### PR TITLE
fix - test issues [ci skip]

### PR DESCRIPTION
**Pull request by ttodua**
(part 1)
these exchanges were failing in tests

### CODE REVIEW

File: test-commonjs.cjs [hunk 1]
```diff
@@ -18,13 +18,13 @@ const symbol = 'BTC/USDT:USDT';
 async function main() {
     try {
         // test cjs version
-        const exchange = new ccxt.bybit({});
+        const exchange = new ccxt.gate({});
```
Comment (ttodua) [inline test-commonjs.cjs:21]: bybit failed too frequently: https://app.travis-ci.com/github/ccxt/ccxt/builds/262664322#L2650

Comment (carlosmiei) [reply test-commonjs.cjs:21]: @ttodua I think this will fix it https://github.com/ccxt/ccxt/pull/17701

Comment (ttodua) [reply test-commonjs.cjs]: ah, havent seen it. ok reverted

Comment (ttodua) [reply test-commonjs.cjs]: btw, it is worth to have bybit there (which needs to have a proxy)? maybe we could replace it with non proxy needed one. like gate or other.

Comment (carlosmiei) [reply test-commonjs.cjs]: i think it is ok, and at least if the proxy stops working we can easily spot it

**Comment by ttodua** [comment]
@carlosmiei passes build.

File: ts/src/lbank2.ts [hunk 2]
```diff
@@ -309,6 +309,8 @@ export default class lbank2 extends Exchange {
                 '3s': true,
                 '5s': true,
             };
+            const amountPrecision = this.parseNumber (this.parsePrecision (this.safeString (market, 'quantityAccuracy')));
+            const contractSize = amountPrecision;
```
Comment (ttodua) [inline ts/src/lbank2.ts:313]: lbank had same size as precision

**Comment by carlosmiei** [comment]
@ttodua question, if these exchanges were failing, shouldn't we remove them from `skip-tests` now?

**Comment by ttodua** [comment]
> @ttodua question, if these exchanges were failing, shouldn't we remove them from `skip-tests` now?

no, these things I've addressed were not failing in current master, but they were failing in my another upcoming stricter-tests PR. so, in-advance I've fixed these cases.

**Comment by ttodua** [comment]
but don't worry, soon after we complete the tests for rest (some few days ahead still) i'll be revising the 'skips' dictionary from again.

**Comment by ttodua** [comment]
let me know if I should change smth here.

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**